articles/agora-cloud-recording/index.md

```diff
@@ -0,0 +1,479 @@
+In this tutorial, we will be building a server using Node.js and Express to acquire recording resource, start, and stop cloud recording of audio/video streams that occur using the Agora SDKs in your application using the Cloud Recording APIs provided by Agora.
```

Comment:
@adrianmurage 
I think this would be technically inaccurate. We are not using the Agora SDKs in this tutorial in any way. It's necessary to mention that we are recording the streams that are occurring in their application.

Reply:
Right...
Noted